pkg/agent/route/route_linux.go

```diff
@@ -324,86 +244,77 @@ func (c *Client) Reconcile(podCIDRs []string) error {
 	}
 	for _, entry := range entries {
 		if desiredPodCIDRs.Has(entry) {
+			klog.Infof("Reconcile has entry %s", entry)
```

Comment:
Do you want to have this log for every entry?

Reply:
removed.